mapclientplugins/filelocationsinkstep/step.py

```diff
@@ -55,12 +45,12 @@ def execute(self):
         """
         # Put your execute step code here before calling the '_doneExecution' method.
         #
-        abs_path = os.path.join(self._location, self._config['file'])
+        abs_path = os.path.join(self._location, self._config['location'])
         for p in self._portData0:
-            dest = shutil.copy(p, abs_path)
             if self._config['prefix'].strip():
-                filename = self._config['prefix'].strip() + '_' + os.path.basename(dest)
-                shutil.move(dest, os.path.join(abs_path,filename))
+                shutil.copy(p, os.path.join(abs_path, self._config['prefix'].strip() + '_' + os.path.basename(p)))
+            else:
+                shutil.copy(p, abs_path)
```

Comment:
This doesn't need to be done for every file; it is constant throughout this method:  self._config['prefix'].strip()

extract out the creation of the modified filename into a simple function and then you should have:

shutil.copy(p, os.path.join(abs_path, _prefix_filename(p, prefix) if prefix else '')